### PR TITLE
chore(framework, novu): Release new versions

### PR DESCRIPTION
**Pull request by rifont**
### What changed? Why was the change needed?

* Release `@novu/framework`@[2.3.0](https://www.npmjs.com/package/@novu/framework/v/2.3.0)
  * Add support for `name` and `description` in Workflow options (https://github.com/novuhq/novu/pull/6708)
  * Propagate Bridge server runtime error messages and stack traces to Bridge client (https://github.com/novuhq/novu/pull/6708)
  * Add support for displaying LiquidJS JSON values in content (https://github.com/novuhq/novu/pull/6714)
* Release `novu`@[2.1.0](https://www.npmjs.com/package/novu/v/2.1.0)
  * Add support for custom tunnels in `npx novu dev` command (https://github.com/novuhq/novu/pull/6711)
* Move `novu` package from `packages/cli` to `packages/novu` to reflect package naming in filesystem location

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: packages/novu/.gitignore
Comment (rifont) [inline packages/novu/.gitignore:1]: Beyond here is just refactors to move `cli` to `novu`, plus the package version change for `novu`.

File: libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts [hunk 1]
```diff
@@ -131,11 +131,13 @@ export class ExecuteBridgeRequest {
...
-       * as it's common for developers to use self-signed certificates in local environments.
-       */
-      rejectUnauthorized: environment.name.toLowerCase() === 'production',
+      https: {
+        /*
+         * Reject self-signed and invalid certificates in Production environments but allow them in Development
+         * as it's common for developers to use self-signed certificates in local environments.
+         */
+        rejectUnauthorized: environment.name.toLowerCase() === 'production',
+      },
```
Comment (rifont) [inline libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts:140]: Fixes this error:
image

**Review by SokratisVidros — APPROVED** [review]
😍

File: libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts (hunk 1 above)
Comment (SokratisVidros) [reply libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts:140]: Nice, I never saw that.

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -3314,118 +3314,6 @@ importers:
         specifier: 5.6.2
         version: 5.6.2
 
-  packages/cli:
```
Comment (rifont) [inline pnpm-lock.yaml:3317]: Makes the necessary symlinking changes as part of moving `packages/cli` -> `package/novu`, plus a few minor dependency updates.

**Pull request merged**